octopus_sensing/tests/test_camera_streaming.py

```diff
@@ -0,0 +1,134 @@
+import os
+import time
+import tempfile
+import queue
+import threading
+import pytest
+
+import octopus_sensing.devices.camera_streaming as camera_streaming
+from octopus_sensing.common.message_creators import start_message, stop_message, terminate_message
+
+class MockedCv2Module:
+
+    def VideoCapture(self, camera_number):
+        return MockedVideoCaptureModule(camera_number)
+    
+    def VideoWriter_fourcc(self, a, b, c, d):
+        return ('XVID')
+    
+    def VideoWriter(self, file_name, codec, fps, _video_size):
+        return MockedVideoWriterModule(file_name)
+
+    '''
+    Get the information of your computer using:
+    >>> import cv2
+    >>> cap=cv2.VideoCapture(0)
+    >>> print(cap.get(cv2.CAP_PROP_FRAME_WIDTH))
+    >>> print(cap.get(cv2.CAP_PROP_FRAME_HEIGHT))
+    '''
+    def CAP_PROP_FRAME_WIDTH(self):
+        return (float(640.0))
+    
+    def CAP_PROP_FRAME_HEIGHT(self):
+        return (float(480.0))
+    
+    def imwrite(self, filename, img):
+        a = time.time()
+        # self.output_path/self.name-experiment_id-str(message.stimulus_id).zfill(2)/str(a).jpg
+        open(filename[:-4] + "/" + str(a) + ".jpg", 'w+').write(img)
+
+class MockedVideoCaptureModule:
+
```

Comment:
There's need to be a `__init__` method. See `VideoCapture` method above. We called this, passing an argument to it. So, it needs an `__init__` method that accepts one argument:

```
    def __init__(self, camera_number):
        pass
```